apps/seasons-sunray-angle/seasons-sunray-angle.jsx

```diff
@@ -63,12 +72,16 @@ export default class SeasonsSunrayAngle extends React.Component {
   }
 
   handleLeapFrame(frame) {
+    if (!this.state.leapConnected) {
+      this.setState({ leapConnected: true });
+    }
     const data = this.gesturesHelper.processLeapFrame(frame);
     if (this.state.activeRaysView === 'space') {
       this.handleSpaceViewGestures(data);
     } else { // ground view
       this.handleGroundViewGestures(data);
     }
+    this.applyOverlayStyle();
```

Comment:
Probably we could call this function in `activeViewPanelChanged` callback instead (and get rid of `activeViewPanel` state).